VimeoNetworkingExample-iOS/VIMUser/user_alum.json

```diff
@@ -1,10 +1,18 @@
 {
     "uri": "mock_uri",
-    "account": "basic",
-    "badge": {
-        "type": "alum",
-        "text": "Alum",
-        "alt_text": "mock_alt_text",
-        "url": "mock_url"
+    "membership": {
+        "type": "basic",
+        "display": "Basic",
```

Comment:
Should this say `Alum`?

Reply:
I think this is correct, I believe the account type is still supposed to be Basic but this type of account would have an Alum badge.